apps/condo/lang/ru/ru.json

```diff
@@ -1734,7 +1734,14 @@
   "notification.messages.MARKETPLACE_INVOICE_CASH_PUBLISHED_MESSAGE.push.title": "Ваш счет за платные услуги \uD83D\uDD14",
   "notification.messages.MARKETPLACE_INVOICE_CASH_WITH_TICKET_PUBLISHED_MESSAGE.push.title": "Ваш счет за платные услуги \uD83D\uDD14",
   "notification.messages.TICKET_CREATED.telegram.openTicket": "Открыть заявку",
+  "notification.messages.TICKET_CREATED.telegram.ticketFilesCount": "(прикреплено файлов: {ticketFilesCount})",
   "notification.messages.TICKET_COMMENT_CREATED.telegram.openTicket": "Открыть заявку",
+  "notification.messages.TICKET_COMMENT_CREATED.telegram.commentFilesCount": "(прикреплено файлов: {commentFilesCount})",
+  "notification.messages.TICKET_COMMENT_CREATED.telegram.ticketFilesCount": "(прикреплено файлов: {ticketFilesCount})",
```

Comment:
```suggestion
```
is ticket files count need to be presented in ticket comment Message?

Reply:
These translations need to be deleted. We cannot correctly count the number of attached files

Reply:
fixed